app/views/layouts/user_mailer.html.slim

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+html xmlns="http://www.w3.org/1999/xhtml" 
+  head
+    = stylesheet_link_tag "email/email"
+    meta content="text/html; charset=utf-8" http-equiv="Content-Type"
+    meta content="width=device-width" name="viewport"
+    title
+      = message.subject
+  body
+    table.body data-made-with-foundation=""
+      tr
+        td.center align="center" valign="top" 
+          center data-parsed="" 
+            table.spacer.float-center
+              tbody
+                tr
+                  td height="10px" style="font-size:10px;line-height:10px;"   
```

Comment:
These are for creating vertical spacers in the email - since the size is arbitrary there really isn't a way to create a class for them unless we make a few utility classes with predefined sizes (.vs10, .vs25, etc) the problem is that I believe the td height tag needs to be explicitly set as well.  Reference: http://foundation.zurb.com/emails/docs/spacer.html